.github/workflows/unit-tests.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Setup environment
         run: |
           sudo apt update
-          sudo apt install -yyq gcc-7 g++-7 clang-6.0 libsystemd-dev gcovr libyaml-dev
+          sudo apt install -yyq gcc-7 g++-7 clang-6.0 libsystemd-dev gcovr libyaml-dev libedit-dev
```

Comment:
Do we need this dependency in the main builds under `packaging`?

Reply:
Ah, this line tries to fill dependencies for flb-wamrc that is AOT compiler using LLVM technology. Now, this should be disabled and no need to add. Should I revert it?

Reply:
I removed it from the workflow fr now.